requirements/requirements-ci.txt

```diff
@@ -57,7 +57,7 @@ flask-cors==3.0.8
 flask-restful==0.3.7
 flask==1.0.3
 frozendict==1.2
-gevent==1.3.7
+gevent==1.5a3
```

Comment:
Is it a good idea to upgrade to an alpha version of 1.5? why not choose the latest `1.4.x`?

Reply:
It it what was proposed in the reference.